src/main/pages/HomePageOnet.java

```diff
@@ -55,30 +52,35 @@ public String acceptCookies2() {
         return getLogoText();
     }
 
-    public void getArticleLinksOpenThemInDifferentTabsAndSaveLinkstoFile() throws IOException {
+    public void getArticleLinksOpenInTabsAndSavetoFile() throws IOException {
         System.out.println("Saving article links to the list");
-        List<WebElement> phraseSearch2 = driver.findElements(By.partialLinkText("ska"));
-        WebElement q;
-        String k;
+        /**
+         Change "ska" to whatever text you want to search it on www.onet.pl site
+         */
+        List<WebElement> searchByPhrase = driver.findElements(By.partialLinkText("ska"));
+        WebElement phraseSearch;
+        String singleArticle;
         List<String> articleLinks = new ArrayList<>();
-        for (WebElement webElement : phraseSearch2) {
-            q = webElement;
-            k = q.getAttribute("href");
-            articleLinks.add(k);
+        for (WebElement webElement : searchByPhrase) {
+            phraseSearch = webElement;
+            singleArticle = phraseSearch.getAttribute("href");
```

Comment:
Change on: "String singleArticle = webElement.getAttribute("href");"
WebElement is declared in for loop, String can be here. No need for reassigning variables in code.

Reply:
Nice, change is made (change will appear in the next commit).